packages/preview/modern-technique-report/0.1.0/template/main.typ

```diff
@@ -0,0 +1,23 @@
+#import "@preview/modern-technique-report:0.1.0": *
+
+#show: modern-technique-report.with(
+  title: [Project Name \ Multiline When too Long],
+  subtitle: [
+    *Fourth Edition*, \ by _H.L. Royden_ and _P.M. Fitzpatrick_
+  ],
+  series: [Mathematics Courses \ Framework Series],
+  author: grid(
+    align: left + horizon,
+    columns: 3,
+    inset: 7pt,
+    [*Member*], [B. Alice], [qwertyuiop\@youremail.com],
+    [], [B. Alice], [qwertyuiop\@youremail.com],
+    [], [B. Alice], [qwertyuiop\@youremail.com],
+    [*Advisor*], [E. Eric], [qwertyuiop\@youremail.com],
+  ),
+  date: datetime.today().display("[year] -- [month] -- [day]"),
+  background: image("bg.jpg"),
+  theme_color: rgb(21, 74, 135),
```

Comment:
I recommend to name everything in kebab-case (i.e. `theme-color`) as it is the most commonly used case convention in Typst.